src/porepy/viz/exporter.py

```diff
@@ -895,37 +962,37 @@ def _sort_and_unify_data(
         # routine.
 
         # Aux. method: Transform scalar- to vector-ranged values.
-        def _to_vector_format(
-            value: np.ndarray, grid: Union[pp.Grid, pp.MortarGrid]
-        ) -> np.ndarray:
+        def _to_vector_format(value: np.ndarray, num: int) -> np.ndarray:
```

Comment:
Same comment on `num` as above. Same name should be used here.

Reply:
okay